py-phylo2vec/src/lib.rs

```diff
@@ -30,6 +31,18 @@ fn build_newick(input_ancestry: Vec<(usize, usize, usize)>) -> String {
     newick_string
 }
 
+#[pyfunction]
+fn sample_ordered(n_leaves: usize) -> Vec<usize> {
+    let v = _utils::sample_ordered(n_leaves);
+    v
+}
```

Comment:
```suggestion
#[pyfunction]
#[pyo3(signature = (n_leaves, ordered=true, /), text_signature = "(n_leaves, ordered=True, /)")]
fn sample(n_leaves: usize, ordered: bool) -> Vec<usize> {
    let v;
    match ordered {
        true => {
            v = _utils::sample(n_leaves, _utils::SampleOrdering::Ordered);
        }
        false => {
            v = _utils::sample(n_leaves, _utils::SampleOrdering::NotOrdered);
        }
    }
    v
}
```

Once you've modified the rust code, to ensure that the python has the correct default values, you can override the function signature with PyO3: https://pyo3.rs/main/function/signature#making-the-function-signature-available-to-python